File: platforms/kotlin/examples/core/texture/set_sampler_options.kt
import org.fragmentcolor.*

val renderer = Renderer()
val pixels: ByteArray = byteArrayOf(255.toByte(), 255.toByte(), 255.toByte(), 255.toByte())
val texture = renderer.createTextureWithSize(pixels, Size(1u, 1u))

val opts = SamplerOptions(repeatX = true, repeatY = true, smooth = true, compare = null)
texture.setSamplerOptions(opts)

File: platforms/swift/examples/core/texture/set_sampler_options.swift
import FragmentColor

let renderer = Renderer()
let pixels: [UInt8] = [255, 255, 255, 255]
let texture = try await renderer.createTextureWithSize(pixels: pixels, size: Size(width: 1, height: 1))

let opts = SamplerOptions(repeatX: true, repeatY: true, smooth: true, compare: nil)
texture.setSamplerOptions(opts: opts)